packages/cli/src/bin/index.ts

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env ts-node
+import semver from 'semver';
+import chalk from 'chalk';
+import program from 'commander';
+import { init } from '../lib/init';
+import { log } from '../lib/log';
+import { board } from '../lib/board';
+import { start } from '../lib/start';
+import { devPlugin } from '../lib/devPlugin';
+import { dataset } from '../lib/dataset';
+import { serve } from '../lib/serve';
+import childProcess from 'child_process';
+
+import pkg from '../package.json';
```

Comment:
This might be unnecessary, see https://docs.npmjs.com/misc/scripts#packagejson-vars, NPM will expose the current package version as: `process.env. npm_package_version`.

Reply:
resolved.